### ISSUE TITLE
Any updates on SD3/Flux nodes progress?

### ISSUE THREAD

**Issue by cdb-boop** [brentjohnston/prompt_injection_advanced#1]
With [renewed interest in targeting specific layers for LoRA training](https://www.reddit.com/r/StableDiffusion/comments/1f523bd/good_flux_loras_can_be_less_than_45mb_128_dim/), having this node working with Flux would be great. Since the other related repos seem to be quiet, I was wondering if you'd made any progress?

**Comment by 311-code** [comment]
Yes I've been working on it here and there, it's not fully working yet. Mostly working on cogvideox at the moment. I believe this is wrong targets but unsure, img_in_weight I want to test with an actual image input, or try torch.cat on some of these: ![flux_injection_test2](https://github.com/user-attachments/assets/33875faa-d41e-4f03-95d9-17b03e5045ac) Thanks for the link!
